### PR TITLE
Revert the changes in `event_accumulator`.

### PR DESCRIPTION
**Pull request by yatbear**
Reverts changes in event_accumulator from https://github.com/tensorflow/tensorboard/pull/5998, this caused internal breakage, where users directly access `EventAccumulator` and unpack the events like tuples.

Will make further changes to the `Event` objects later.

Tested internally: cl/483980532 (will wait for the global test to complete before merging this).

#oncall

### CODE REVIEW

File: tensorboard/plugins/hparams/list_session_groups.py [hunk 1]
```diff
@@ -565,7 +565,7 @@ class _MetricIdentifier:
     Attributes:
       group: Metric group corresponding to the dataset on which the model was
         evaluated.
-      tag: String tag assoicated with the metric.
+      tag: String tag associated with the metric.
```
Comment (nfelt) [inline tensorboard/plugins/hparams/list_session_groups.py:568]: For a revert, I'd avoid introducing any unrelated changes, even if they're just a typo fix. Part of the idea w/ a revert is that it shouldn't really require review other than seeing that it's a rollback of a previous change, so it's more important than usual to avoid introducing any unrelated deltas.

**Review by nfelt — APPROVED** [review]

File: tensorboard/plugins/hparams/list_session_groups.py (hunk 1 above)
Comment (yatbear) [reply tensorboard/plugins/hparams/list_session_groups.py]: Got it, reverted the typo fix.
